src/profile.ts

```diff
@@ -239,12 +239,12 @@ export class Profile {
     throw new NotFoundError(`Not able to resolve profile for ${addressOrName}`);
   }
 
-  static async getMulti(addressesOrNames: string[], config: Config): Promise<Profile[]> {
+  static async getMulti(addressesOrNames: string[], config: Config): Promise<(Profile | null)[]> {
```

Comment:
I don't think there's much value in returning `null`s here, is there? Why not filter the results so we only return the projects that resolved?

Reply:
Done in [22c6bbc](https://github.com/radicle-dev/radicle-interface/pull/327/commits/22c6bbc42cae62b40f1f3997a5e271ea02369b9f)

Reply:
The idea behind the `null` was to not filter at this stage but return project or `null`, and then the component can handle what to do if a `null` returns.
But we can also handle the non existence of a array item as `null` 😄